tests/functionnal/RuleTicket.php

```diff
@@ -561,4 +565,96 @@ public function testGRoupRequesterAssignFromDefaultUser() {
          ])
       )->isTrue();
    }
+
+   public function testTaskTemplateAssignFromRule() {
+      $this->login();
+
+      // Create solution template
+      $task_template = new TaskTemplate();
+      $task_template_id = $task_template->add([
+         'content' => "test content"
+      ]);
+      $this->integer($task_template_id);
+
+      // Create rule
+      $rule_ticket_em = new \RuleTicket();
+      $rule_ticket_id = $rule_ticket_em->add($ruletinput = [
+         'name'         => "test to assign ITILSolution",
+         'match'        => 'OR',
+         'is_active'    => 1,
+         'sub_type'     => 'RuleTicket',
+         'condition'    => \RuleTicket::ONADD + \RuleTicket::ONUPDATE,
+         'is_recursive' => 1,
+      ]);
+      $this->integer($rule_ticket_id);
```

Comment:
```suggestion
      $this->integer($rule_ticket_id)->isGreaterThan(0);
```

Same for all add checks